packages/flutter_tools/lib/src/features.dart

```diff
@@ -72,13 +72,19 @@ abstract class FeatureFlags {
     nativeAssets,
     swiftPackageManager,
   ];
+
+  /// All current Flutter feature flags that can be configured.
+  ///
+  /// [Feature.configSetting] is not `null`.
+  Iterable<Feature> get allConfigurableFeatures {
+    return allFeatures.where((Feature feature) => feature.configSetting != null);
```

Comment:
Should we include features that have a `environmentOverride`? I don't know why you'd do this, but you could have a feature that has an `environmentOverride` but not a `configSetting`.

If you move forward with this, don't forget to update the `// SAFETY: allConfigurableFeatures only returns features with a non-null configSetting` bit up above.

If you don't move forward with this, consider updating `Feature`'s constructor to assert that `configSetting` is provided if `environmentOverride` is also provided.